pkg/pipeline/extract/conntrack/multiorderedmap.go

```diff
@@ -0,0 +1,135 @@
+/*
```

Comment:
@ronensc this feels like a generic helper utility, can you move that to `utils` ??

Reply:
Done